apps/src/netsim/NetSimConnection.js

```diff
@@ -201,6 +201,16 @@ NetSimConnection.prototype.onMyNodeChange_= function () {
 };
 
 /**
+ * Detects when local client node is offline.  Responds by kicking the user
+ * out of the shard, which should update the interface and provide an
+ * opportunity to reconnect.
+ * @private
+ */
+NetSimConnection.prototype.onMyNodeLostConnection_ = function () {
+  this.disconnectFromShard();
+};
```

Comment:
Yes, I would like to do more here in the future; but that's the future.  Good call on the cleaner version for this iteration.